Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM python:3.6-alpine
+ENV PYTHONUNBUFFERED 1
+ENV PY_ENV development
+
+RUN mkdir /app
```

Comment:
The `mkdir` is unnecessary with `workdir`, it will be made automatically - you can 🔪 this